cli-rs/src/main.rs

```diff
@@ -113,7 +114,8 @@ enum Commands {
         #[arg(default_value_t = false)]
         continuations: bool,
     },
-    /// Executes a powdr-asm file with given inputs.
+    /// Execute a powdr-asm file with given inputs.
+    /// Does not generate a witness.
     Execute {
```

Comment:
now there are two separate commands: `execute` for running the riscv executor in "fast" non witgen mode, and `witgen` which runs in "trace" mode and generates a valid witness